project/ISSUE-TRIAGE.md

```diff
@@ -1,4 +1,4 @@
-Triaging of issues
+# Triaging of issues
```

Comment:
The `---------` below is the alternative syntax for creating a H2; can you remove that line? (see https://daringfireball.net/projects/markdown/basics)